docs/en/01_Working_with_projects/15_cwp_logging.md

```diff
@@ -0,0 +1,94 @@
+# How to use CWP logging
+
+When trying to identify the source of an issue or problem with your website, you will find yourself wanting to look at the server logs to try and find the root cause of your problem. This guide will help you with the CWP logging system and how to use it to get the information you need.
+
+**Location:** https://logs.cwp.govt.nz/
+
+## Introduction
+
+CWP use Graylog as an internal logging collection and display dashboard. This has a lot of built in functionality that can be used for searching logs which you can find here. http://docs.graylog.org/en/2.4/pages/queries.html
```

Comment:
CWP `uses` Graylog
as in: "Common Web Platform use Graylog" doesn't make much sense :)

Reply:
Maybe "This tool has" as well... as a minor at your discretion :>